exports.go

```diff
@@ -34,14 +34,40 @@ const (
 	ResponseTypeChunked = "Chunked"
 )
 
+// ServiceLatency is used when observing and exported service for
+// latency measurements.
+// Sampling 1-100, represents sampling rate, defaults to 100.
+// Results is the subject where the latency metrics are published.
+// A metric will be defined by the nats-server's ServiceLatency. Time durations
+// are in nanoseconds.
+// see https://github.com/nats-io/nats-server/blob/master/server/accounts.go#L524
+// e.g.
+// {
+//  "app": "dlc22",
+//  "start": "2019-09-16T21:46:23.636869585-07:00",
+//  "svc": 219732,
+//  "nats": {
+//    "req": 320415,
+//    "resp": 228268,
+//    "sys": 0
+//  },
+//  "total": 768415
+// }
+//
+type ServiceLatency struct {
+	Sampling int     `json:"sampling,omitempty"`
```

Comment:
Would it be more self-describing if this was called "SamplingPercentage"

Reply:
Might be, but feels too much like Java ;)